lib/cs_guide/resources/venue.ex

```diff
@@ -9,8 +9,14 @@ defmodule CsGuide.Resources.Venue do
 
   schema "venues" do
     field(:venue_name, :string)
+    field(:description, :string)
+    field(:address1, :string)
+    field(:address2, :string)
     field(:postcode, :string)
+    field(:website, :string)
```

Comment:
`:website` should use `Fields.url` as per: https://github.com/dwyl/fields/issues/16
which will do automatic validation & sanitisation. creating it here using a `:string` means we are going to duplicate effort unnecessarily.